tests/trnsysGUI/diagram/data/examples/TRIHP_dualSource/expected/TRIHP_dualSource.dck

```diff
@@ -0,0 +1,4065 @@
+CONSTANTS 1
+unitPrintEBal=164
+ASSIGN temp\ENERGY_BALANCE_MO.Prt unitPrintEBal
+UNIT 164 Type 46
+PARAMETERS 6
+unitPrintEBal !1: Logical unit number
+-1 !2: for monthly summaries
+1  !3: 1:print at absolute times
+-1 !4 -1: monthly integration
+1  !5 number of outputs to avoid integration
+1  !6 output number to avoid integration
+INPUTS 29
+TIME elSysOut_CtrlDemand elSysOut_BatLoss elSysIn_Grid elSysOut_PvToGrid elSysOut_InvLoss PelLossesDCAC elSysIn_PV qSysIn_Ghx elSysIn_Q_HpComp elSysOut_PuCond elSysOut_PuEvap qSysOut_PipeLoss qSysOut_spPipeIntTot qSysOut_BuiDemand	 qSysIn_BuiDemand	 elSysOut_PuShDemand	 qSysOut_DhwDemand elSysOut_HHDemand qSysOut_TesDhwLoss qSysOut_TesDhwAcum elSysIn_Q_TesDhwAux qSysOut_TesShLoss qSysOut_TesShAcum elSysIn_Q_TesShAux qSysOut_CiBuiSH qSysOut_CiBui qSysOut_CiLoss qImb
+*******************************
+TIME elSysOut_CtrlDemand elSysOut_BatLoss elSysIn_Grid elSysOut_PvToGrid elSysOut_InvLoss PelLossesDCAC elSysIn_PV qSysIn_Ghx elSysIn_Q_HpComp elSysOut_PuCond elSysOut_PuEvap qSysOut_PipeLoss qSysOut_spPipeIntTot qSysOut_BuiDemand	 qSysIn_BuiDemand	 elSysOut_PuShDemand	 qSysOut_DhwDemand elSysOut_HHDemand qSysOut_TesDhwLoss qSysOut_TesDhwAcum elSysIn_Q_TesDhwAux qSysOut_TesShLoss qSysOut_TesShAcum elSysIn_Q_TesShAux qSysOut_CiBuiSH qSysOut_CiBui qSysOut_CiLoss qImb
+***************************************************************
+**BEGIN Hourly Energy Balance printer automatically generated from DDck files
+***************************************************************
+CONSTANTS 1
+unitPrintEBal_h=165
+ASSIGN temp\ENERGY_BALANCE_HR.Prt unitPrintEBal_h
+UNIT 165 Type 46
+PARAMETERS 6
+unitPrintEBal_h !1: Logical unit number
+-1 !2: for monthly summaries
+1  !3: 1:print at absolute times
+1 !4 1: hourly integration
+1  !5 number of outputs to avoid integration
+1  !6 output number to avoid integration
+INPUTS 29
+TIME elSysOut_CtrlDemand elSysOut_BatLoss elSysIn_Grid elSysOut_PvToGrid elSysOut_InvLoss PelLossesDCAC elSysIn_PV qSysIn_Ghx elSysIn_Q_HpComp elSysOut_PuCond elSysOut_PuEvap qSysOut_PipeLoss qSysOut_spPipeIntTot qSysOut_BuiDemand	 qSysIn_BuiDemand	 elSysOut_PuShDemand	 qSysOut_DhwDemand elSysOut_HHDemand qSysOut_TesDhwLoss qSysOut_TesDhwAcum elSysIn_Q_TesDhwAux qSysOut_TesShLoss qSysOut_TesShAcum elSysIn_Q_TesShAux qSysOut_CiBuiSH qSysOut_CiBui qSysOut_CiLoss qImb
+*******************************
+TIME elSysOut_CtrlDemand elSysOut_BatLoss elSysIn_Grid elSysOut_PvToGrid elSysOut_InvLoss PelLossesDCAC elSysIn_PV qSysIn_Ghx elSysIn_Q_HpComp elSysOut_PuCond elSysOut_PuEvap qSysOut_PipeLoss qSysOut_spPipeIntTot qSysOut_BuiDemand	 qSysIn_BuiDemand	 elSysOut_PuShDemand	 qSysOut_DhwDemand elSysOut_HHDemand qSysOut_TesDhwLoss qSysOut_TesDhwAcum elSysIn_Q_TesDhwAux qSysOut_TesShLoss qSysOut_TesShAcum elSysIn_Q_TesShAux qSysOut_CiBuiSH qSysOut_CiBui qSysOut_CiLoss qImb
+*******************************
+**BEGIN Head.ddck
```

Comment:
@zuckerruebe: please clean up End.ddck

Reply:
Not in this pull request. This has been around for a long time.